system/kube-monitoring/charts/prometheus-frontend/README.md

```diff
@@ -33,20 +33,26 @@ Region is an implicit label that is automatically set by Prometheus. Do not over
    * `openstack`
    * `datapath`
    * `kubernikus`
+   * `e2e`
```

Comment:
Why are you introducing the `e2e` tier? Currently we're using `tier=datapath, context=availability` for these tests.

Reply:
Can't follow based on your explanation. The tier label doesn't change the what/why, that's up to the respective test to do that. But okay if this was already decided.